src/reynir/verbframe.py

```diff
@@ -210,15 +213,19 @@ class VerbFrame:
     def __init__(
         self,
         verb: str,
+        obj: str,
+        iobj: str,
         args: List[str],
         preps: Iterable[Tuple[str, str]],
         particle: Optional[str],
         score: Optional[int],
     ) -> None:
         self.verb = verb
-        assert 0 <= len(args) <= 2
+        #assert 0 <= len(args) <= 2
```

Comment:
Move this assert to below the `self.cases` assignment below and do `assert 0 <= len(self.cases)<= 2`

Reply:
Done